src/theme/styles/colours.ts

```diff
@@ -0,0 +1,53 @@
+export const BASE_COLOURS = {
```

Comment:
I only see these colors used on storybook.
Are they not used anywhere else?

Reply:
They will be used later, yes